index.html

```diff
@@ -0,0 +1,335 @@
+            >
+          </div>
+        </div>
+      </div>
+      <div class="about-me">
+        <section class="about-me-section">
+          <h2>About me</h2>
+          <p class="about-me-text">
+            I am a skilled Full-Stack Web Developer who loves making pages with
+            beautiful UI. I have studied at Microverse, and I am experienced in
+            remote work.
+            <br />
+            I've also been a good helping hand to a lot of developers in
+            different communities!
+            <br />
+            if you'd like to get in contact about potential job opportunities,
+            please reach out!
+          </p>
+          <a
+            href="https://docs.google.com/document/d/1lYq0Kb50TujzJzDWGobIchRI17JMnAiZMF9OBQgNU68/edit?usp=sharing"
+            class="link"
+            target="_blank"
+            rel="noreferrer noopener"
+          >
+            Get my resume
+          </a>
+          <img class="about-me-icon" src="./media/svg/icon-2.svg" alt="Icon" />
+        </section>
+        <hr />
+        <div class="about-me-section stacks">
+          <ul>
+            <li class="languages list-title">
+              Languages
+              <ul>
+                <li>JavaScript</li>
+                <li>TypeScript</li>
+                <li>Ruby</li>
+                <li>HTML</li>
+                <li>CSS</li>
+                <li>Sass</li>
+              </ul>
+            </li>
+          </ul>
+          <ul>
+            <li class="frameworks list-title">
+              Frameworks
+              <ul>
+                <li>React & Redux</li>
+                <li>Material-UI</li>
+                <li>Jest</li>
+                <li>Bootstrap</li>
+                <li>Ruby on Rails</li>
+                <li>RSPec</li>
+              </ul>
+            </li>
+          </ul>
+          <ul>
+            <li class="skills list-title">
+              Skills
+              <ul>
+                <li>GitHub</li>
+                <li>CodePen</li>
+                <li>GitLab</li>
+                <li>Terminal</li>
+                <li>WSL</li>
+                <li>Remote Pair-Programming</li>
+                <li>Teamwork</li>
+                <li>Mentoring</li>
+                <li>Code Reviews</li>
+                <li>Mobile/Responsive Development</li>
+              </ul>
+            </li>
+          </ul>
+        </div>
+      </div>
+      <div class="form-section">
+        <p>
+          <b class="form-description">
+            I'm always interested in hearing about new projects, so if you'd
+            like to chat please get in touch.</b
+          >
+        </p>
+        <form
+          class="form"
+          action="https://formspree.io/f/maykyqpv"
+          method="POST"
+        >
+          <input
+            type="text"
+            class="user-input"
+            placeholder="Full name"
+            maxlength="30"
+            name="user-name"
+          />
```

Comment:
Please include a `required` attribute to indicate that the user must specify a value. This comment is applicable to **all the input fields** for this project. The reason behind this is that whenever the user leaves an empty field, the form can still be submitted. You can refer to the [link](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/required).

Using `required` attribute will activate the validation on submit (i.e. *when user clicks the submit button*)
![image](https://user-images.githubusercontent.com/98527559/184050299-c945e5f3-9d1d-486c-bea2-3818f5aa4570.png)


- [x] You can apply this as an **inline style property** like so: 
```
<input type="indicate_type_of_input_here" required="required">
```
OR simply
```
<input type="indicate_type_of_input_here" required>
```